docs/modules/commons/pages/table-transformers.adoc

```diff
@@ -516,3 +516,96 @@ Examples:
 |value |1    |
 |value2|0    |
 ----
+
+== INNER_JOIN
+
+`INNER_JOIN` transformer combines rows from two tables whenever there are matching values between the columns.
+
+[cols="1,3", options="header"]
+|===
+|Parameter
+|Description
+
+|`leftTableJoinColumn`
+|the column name for matching in the first (left) table
+
+|`rightTableJoinColumn`
+|the column name for matching in the second (right) table
+
+|`tables`
+|paths to examples tables or transformers to inner join
+|===
+
+NOTE: The table is called `left`, when it specified using the first path to xref:ROOT:glossary.adoc#_examplestable[ExamplesTable] or nested transformer in `tables` parameter. Otherwise the table is called `right` (the second path, the second nested transformer or specified as body under transformer)
+
+[IMPORTANT]
+====
+* number of unique tables paths (nested transformers) must be equal 1 if one table is specified as body under transformer, otherwise it must be equal 2
```

Comment:
even more complicated then it was before, not clear what is said here

Reply:
changed